components/lwip/Kconfig

```diff
@@ -90,6 +90,13 @@ menu "LWIP"
         help
             Enabling this option allows checking for available data on a netconn.
 
+    config LWIP_NETBUF_RECVINFO
```

Comment:
I would request this to be separate PR, unless it has dependency on other fix (i.e. build issues with mbedtls client only mode configuration).

Reply:
Ok - will split it later today when I get back to the office